src/widgets/filepickerwidget.h

```diff
@@ -57,18 +57,14 @@ public slots:
     //! The selected state of some paths in the underlying model has changed.
     void selectionChanged();
 
-    //! The focus is no longer on this widget.
-    void focusLost();
-
     //! One of the setting checkboxes has changed.
     void settingChanged();
 
 protected:
     //! Used for handling the ESC key.
     void keyPressEvent(QKeyEvent *event);
 
-    //! Used for determining if the focus was lost, and then emitting
-    //! \ref focusLost and saving the currently-browsed path.
+    //! Used for saving the currently-browsed path if the focus was lost.
```

Comment:
The last browsed location functionality should be moved to FilePickerDialog where it's actually useful, it doesn't make sense to be in this base widget, think of the Jobs file browser. The serialization should also occur in the destructor, instead of the focusOutEvent.